.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,29 @@
 ##### SUMMARY
-<!--- Describe the change, including rationale and design decisions -->
+<!--- Describe the change below, including rationale and design decisions -->
```

Comment:
@dagwieers @gundalow why didn't we remove summary title as was suggested in the original version of PR description? (can't find this decision in comments)

Reply:
Because I think you raised the issue that those might be used by @ansibot. So I removed everything that may be unsafe.

I still have my original proposal and want to do a second PR with stuff that is harder to get consensus of, and requires changes to @ansibot or other tools.